tsconfig.json

```diff
@@ -22,5 +22,15 @@
 			"*": [ "*", "client/*", "server/*" ]
 		}
 	},
-	"include": [ "client", "server" ]
+	"include": [ "client", "server" ],
+	"exclude": [
```

Comment:
I'd add these "build" directories:

```
apps/**/dist
packages/**/dist
```

You can generate these folders by running `npx lerna run build` for packages and `npx lerna run prepare` for packages.

Can `./cached-requests.json` get big?

Add `./stats.json` (used for is-calypso-fast-yet)?

Add `/public`?

Reply:
We could add `apps/*/dist` but i'd avoid `packages/*/dist`. Some packages are built and have a package.json that points into their `dist` directories, so we probably don't want to exclude those.

Reply:
I added these all to the excludes.

I'm not worried about cutting out `packages/*/dist` because TypeScript shouldn't be getting anything meaningful from them. correct me if I'm wrong.

this is all about helping `tsc` quickly get type information from dependencies and I'd probably rather _not_ get limited inference from generated bundles than have it spend its time working through those large and difficult files.

thanks for the suggestions